manifests/cf-manifest/operations.d/320-cc-increase-broker-client-timeout.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Increase the timeout duration for brokers
+# Specifically it can take a long time to create cloudfront CDN distributions
+
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/broker_client_timeout_seconds?
+  value: 300
+
+- type: replace
+  path: /instance_groups/name=cc-worker/jobs/name=cloud_controller_worker/properties/cc/broker_client_timeout_seconds?
+  value: 300
+
+- type: replace
```

Comment:
I don't see this property in the job config: https://github.com/cloudfoundry/capi-release/blob/develop/jobs/cloud_controller_clock/spec